ci-operator/config/red-hat-data-services/ods-ci/red-hat-data-services-ods-ci-master__rhoai2.16-ocp4.18-lp-interop.yaml

```
@@ -0,0 +1,74 @@
+base_images:
+  cli:
+    name: "4.18"
+    namespace: ocp
+    tag: cli
+  rhoai-tests:
+    name: ods-ci
+    namespace: ci
+    tag: dev
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.18"
+resources:
+  '*':
+    limits:
+      cpu: "8"
+      memory: 32Gi
+    requests:
+      cpu: "8"
+      memory: 32Gi
+tests:
+- as: rhoai-interop-aws
+  cron: 0 0 11 6 *
+  steps:
+    cluster_profile: aws-rhoai-qe
+    env:
+      BASE_DOMAIN: rhods.ccitredhat.com
```

```suggestion
      BASE_DOMAIN: aws.rh-ods.com
```
Sorry for misleading previously, this is the base domain we should use.